user_interface:
  ros__parameters:
    ingredient_id: 3
    robot_position: [12.3, 20.8]
    stove_level: "Max"
    keep_temperature_sec: 5
